RUFAS/routines/manure/manure_treatments/base_manure_treatment.py

```diff
@@ -334,12 +337,25 @@ def _get_current_day_rainfall(self) -> float:
         precipitation = current_conditions.precipitation
         return precipitation * GeneralConstants.MM_TO_M
 
-    def _accumulate_daily_output(self, manure_treatment_daily_output: ManureTreatmentDailyOutput) -> None:
-        """Accumulates the daily output of the manure treatment.
+    def _adjust_accumulated_output(self, manure_treatment_daily_output: ManureTreatmentDailyOutput) -> None:
+        """Adjust the accumulated output by either resetting it or adding the daily output to it.
+        The accumulated output will be reset on the first day of every storage time period.
 
```

Comment:
```suggestion
        
        Notes
        -----
        The accumulated output will be reset on the first day of every storage time period.
```
Would it make more sense to have it be reset on the final day of every storage time period? I don't think it makes a very big difference, but this would prevent the manure treatment from being reset on the very first day of the simulation. If so, probably an issue for a different PR.

Reply:
It's in an interesting thought. I don't know if it's helpful to have the data or clear reset on the first simulation day? @tomhuhh do you have any opinion on the need for the emptying to occur on the first day of every storage time period or at the end? It's something I'd hesitate to change without knowing the reason why it was done like this in the first place in case there is a good reason the emptying should occur on the first day of the simulation.

Reply:
Presently I think there is no accumulated manure in any manure treatment at the start of the simulation, so there would be nothing to empty if we did empty on the first day of the simulation. Emptying also does not necessarily make the value of accumulated manure = 0, it just utilizes 0 as the value for _yeserday's_ accumulated manure. So the accumulated manure in storage on the emptying day would be 0 + that day's manure entering storage. I do think that the emptying should occur on the first day of the storage time period though, otherwise we would end up with a functional storage time period of n - 1 for the first storage time interval.